src/js/claims-status/containers/AppealInfo.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AppealsV2StatusPage from './AppealsV2StatusPage';
+import AppealsV2DetailPage from './AppealsV2DetailPage';
+import AppealsV2TabNav from '../components/appeals-v2/AppealsV2TabNav';
+
+const PATHS = { status: 'status', detail: 'detail' };
+
+const setActiveTab = (currentPath, paths = { status: '', detail: '' }, appealId) => {
+  if (currentPath.includes(paths.status)) {
+    return <AppealsV2StatusPage appealId={appealId}/>;
+  } else if (currentPath.includes(paths.detail)) {
+    return <AppealsV2DetailPage appealId={appealId}/>;
```

Comment:
🤔  I'm not super keen on this bit; see my comment in `routes.jsx`. I think we can get around this with a bit more fancy footwork in `react-router`.

Reply:
I decided to keep things consistent-ish with what was done for Claims before but...I think I like the router-based implementation better. Will play around and see what I can do.